group-coordinator/src/main/java/org/apache/kafka/coordinator/group/metrics/GroupCoordinatorRuntimeMetrics.java

```diff
@@ -120,19 +121,14 @@ public GroupCoordinatorRuntimeMetrics(Metrics metrics) {
                 "The average time it took to load the partitions in the last 30 sec."
             ), new Avg());
 
-        this.threadIdleRatioSensor = metrics.sensor("ThreadIdleRatio");
-        this.threadIdleRatioSensor.add(
-            metrics.metricName(
-                "thread-idle-ratio-min",
+        this.threadIdleSensor = metrics.sensor("ThreadIdleRatio");
+        this.threadIdleSensor.add(new Meter(TimeUnit.MILLISECONDS,
+            metrics.metricName("thread-idle-ratio",
                 METRICS_GROUP,
-                "The minimum thread idle ratio over the last 30 seconds."
-            ), new Min());
-        this.threadIdleRatioSensor.add(
-            metrics.metricName(
-                "thread-idle-ratio-avg",
+                "The fraction of time the threads spent waiting for an event."),
+            metrics.metricName("thread-idle-time-ms-total",
```

Comment:
I am not sure to understand the value brought by this one. Does it bring any value?

Reply:
Moved to using a Rate instead of Meter